tests/testthat/test-api-endpoints.R

```diff
@@ -331,44 +331,6 @@ test_that("strategise", {
   db <- mintr_test_db()
   res <- target_strategise(db)(json)
   expect_is(res, "json")
-  expect_equal(res, jsonlite::toJSON(
```

Comment:
Taking out testing for specific data here?

Reply:
This changed and there were no notes about how the values were derived. The underlying optimisation functions do look to have tests, but this would just be churn